src/server/passive_tran_server.cpp

```diff
@@ -150,12 +153,33 @@ void passive_tran_server::send_and_receive_stop_log_prior_dispatch ()
   // needs to be consumed (aka: waited to be consumed/serialized to log)
 }
 
-void passive_tran_server::send_oldest_active_mvccid ()
+void passive_tran_server::start_oldest_active_mvccid_sender ()
+{
+  assert (m_oldest_active_mvccid_sender == nullptr);
+
+  /* Now 1s , but it would be a system parameter later to make it tunable. */
+  cubthread::looper loop (std::chrono::milliseconds (1000));
+  auto func_exec = std::bind (&passive_tran_server::send_oldest_active_mvccid, std::ref (*this), std::placeholders::_1);
+  auto sender_entry = new cubthread::entry_callable_task (std::move (func_exec)); /* delete on retire. See the constr. */;
+
+  m_oldest_active_mvccid_sender = cubthread::get_manager ()->create_daemon (loop, sender_entry,
+				  "passive_tran_server::oldest_active_mvccid_sender");
+
+  assert (m_oldest_active_mvccid_sender != nullptr); // when create_daemon() fails
```

Comment:
No error code is defined for this case. I actually was not sure I should put the assert here since no other caller even doesn't check if it succeeds or not, but I did for safety. Do you think we should define an error for this?

Reply:
No, just leave it like this.